django-spectre-css/templates/spectre-css/form.html

```diff
@@ -0,0 +1,18 @@
+{% load widget_tweaks %}
+
+{%  csrf_token %}
+{% for hidden_field in form.hidden_fields %}
+  {{ hidden_field }}
+{% endfor %}
+
+{% if form.non_field_errors %}
+  <div class="alert alert-danger" role="alert">
```

Comment:
Does Spectre not call these classes `toast` rather than `alert`?

Reply:
yes, my custom css fooled me here